spec/todolist.spec.js

```diff
@@ -1 +1,77 @@
+const {
+  createTodo,
+  todos,
+  getAllTodos,
+  setComplete,
+  searchById,
+  filterByComplete,
+  removeTodo
+} = require('../src/todolist')
 
+describe('todos', () => {
+  beforeEach(() => {
+    todos.splice(0, todos.length)
+  })
+
+  it('createToDo should add an object to todos', () => {
+    createTodo('Watch world cup')
+
+    expect(todos).toEqual([{ task: 'Watch world cup', complete: false, id: 1 }])
```

Comment:
Does this test pass? The `createTodo` function returns an object, but your test expects it to return an array containing an object

Reply:
It passes, I imported the whole todos list and was checking if the created todo item was successfully pushed into the main list. Not sure if it's a good thing to test for though